Microsoft.Azure.Cosmos/src/Telemetry/OpenTelemetry/OpenTelemetryDefaultRecorder.cs

```diff
@@ -0,0 +1,83 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace Microsoft.Azure.Cosmos.Telemetry.Diagnostics
+{
+    using System;
+    using System.Net;
+    using global::Azure.Core.Pipeline;
+    using Microsoft.Azure.Cosmos.Diagnostics;
+    using Microsoft.Azure.Cosmos.Tracing;
+
+    internal sealed class OpenTelemetryDefaultRecorder : IOpenTelemetryRecorder
```

Comment:
nit: The standard being followed is to use Core instead of default
```suggestion
    internal sealed class OpenTelemetryCoreRecorder : IOpenTelemetryRecorder
```